soc/espressif/common/Kconfig.esptool

```
@@ -136,22 +135,22 @@ choice ESPTOOLPY_FLASHSIZE
 	help
 	  SPI flash size, in megabytes
 
-	config ESPTOOLPY_FLASHSIZE_1MB
-		bool "1 MB"
-	config ESPTOOLPY_FLASHSIZE_2MB
-		bool "2 MB"
-	config ESPTOOLPY_FLASHSIZE_4MB
-		bool "4 MB"
-	config ESPTOOLPY_FLASHSIZE_8MB
-		bool "8 MB"
-	config ESPTOOLPY_FLASHSIZE_16MB
-		bool "16 MB"
-	config ESPTOOLPY_FLASHSIZE_32MB
-		bool "32 MB"
-	config ESPTOOLPY_FLASHSIZE_64MB
-		bool "64 MB"
-	config ESPTOOLPY_FLASHSIZE_128MB
-		bool "128 MB"
+config ESPTOOLPY_FLASHSIZE_1MB
+	bool "1 MB"
+config ESPTOOLPY_FLASHSIZE_2MB
+	bool "2 MB"
```

Newlines between config items (also for the rest of the PR)
```suggestion
config ESPTOOLPY_FLASHSIZE_1MB
	bool "1 MB"

config ESPTOOLPY_FLASHSIZE_2MB
	bool "2 MB"
```